caiman/gui/gui_pyqtgraph_layout.py

```diff
@@ -46,32 +49,39 @@ def make_color_img(img, gain=255, min_max=None,out_type=np.uint8):
 # cnm_obj = load_CNMF('/Users/agiovann/caiman_data/example_movies/memmap__d1_60_d2_80_d3_1_order_C_frames_2000_save.hdf5')
 cnm_obj = load_CNMF(F.getOpenFileName(caption='Load CNMF Object',filter='*.hdf5')[0])
 
-
 # movie
 # mov = cm.load('/Users/agiovann/caiman_data/example_movies/memmap__d1_60_d2_80_d3_1_order_C_frames_2000_.mmap')
 mov = cm.load(cnm_obj.mmap_file)
 min_mov = np.min(mov)
 max_mov = np.max(mov)
-
-# load summary image
-# Cn = cm.load('/Users/agiovann/caiman_data/example_movies/memmap__d1_60_d2_80_d3_1_order_C_frames_2000__Cn.tif')
-Cn = cnm_obj.estimates.Cn
-
-
+mode = 'reset'
 estimates = cnm_obj.estimates
-min_mov_denoise = np.min(estimates.A.dot(estimates.C))
-max_mov_denoise = np.max(estimates.A.dot(estimates.C))
-background_num = -1
-neuron_selected = False
-nr_index = 0
+bg = np.dot(estimates.b.reshape([estimates.dims[0],estimates.dims[1],-1], order='F'), estimates.f).transpose([2,0,1])
```

Comment:
I would avoid constructing this as it can run out of memory quickly for large datasets.